thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dataframe/util/DataFrameUtils.java

```diff
@@ -147,17 +147,12 @@ public static DataFrame evaluateExpressions(DataFrame df, Collection<MetricExpre
    * @return dataframe with modified timestamps
```

Comment:
Fix comment? Seems no longer does filling NULL values for missing rows.

Reply:
good catch. fixed